app/src/main/java/com/github/fribourgsdp/radio/GoogleSignInActivity.kt

```diff
@@ -100,8 +100,8 @@ class GoogleSignInActivity : AppCompatActivity() {
             //check if user is new or existing
             if (authResult.additionalUserInfo!!.isNewUser) {
                 //user is new - Account Create
-                val mail = saveUserInDatabse(authResult)
-                saveTestUser(this, mail)
+               // val mail = saveUserInDatabse(authResult)
+                //saveTestUser(this, mail)
```

Comment:
Why has this been removed? Won't this disable @Flamantrose321 's feature of having the newly logged in user's initials in his/her profile picture?

Reply:
See my answer to @Flamantrose321 as why I don't think we should do this here.